src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/MaturedBlocksSyncManager.cs

```diff
@@ -33,7 +33,7 @@ public class MaturedBlocksSyncManager : IMaturedBlocksSyncManager
         private Task blockRequestingTask;
 
         /// <summary>The maximum amount of blocks to request at a time from alt chain.</summary>
-        private const int MaxBlocksToRequest = 1000;
+        private const int MaxBlocksToRequest = 100;
```

Comment:
Reason for changing this is running sidechains on btc. With 1k blocks we will transfer 1gb of data. That's something we will usually not be able to do in 60 sec. 

Also it prevents the same problem on sidechains which might become quite heavy if there are many ppl using smart contracts. 
Also I think in the future this should be changed to maxBytes limitation instead of count of blocks.

Reply:
Not sure about this change. I understand the motivation behind it but for now, we don't peg to bitcoin and we're going to make 10x more calls as a result (at least at the beginning).